src/components/Integer/IntegerSidebar.jsx

```diff
@@ -33,6 +45,47 @@ const IntegerSidebar = (props) => {
           }}
         />
       </Segment>
+      <Segment className="form sidebar-image-data">
+        <TextWidget
+          id="external"
+          title={props.intl.formatMessage(messages.placeholder)}
+          value={props.data.placeholder}
+          onChange={(e, v) => {
```

Comment:
please try to avoid single-character vars except for trivial things like "i", "j" when iterating. Use key, value instead.

Reply:
resolved